README.md

```
@@ -14,24 +14,29 @@ ex) `week2/programmers: 폰켓몬`
-|폰켓몬|⭐||
-|전화번호 목록|⭐⭐||
-|의상|⭐⭐||
-|베스트앨범|⭐⭐⭐|https://programmers.co.kr/learn/courses/30/lessons/42579|
+|같은 숫자는 싫어|⭐|
+|기능 개발|⭐⭐|https://programmers.co.kr/learn/courses/30/lessons/42586|
+|올바른 괄호|⭐⭐|
+|프로세스|⭐⭐|
+|다리를 지나는 트럭|⭐⭐|
+|주식 가격|⭐⭐|https://programmers.co.kr/learn/courses/30/lessons/42584|
+
 ####  👊 추가 문제
```

markdownlint-cli2 (0.17.2)

Multiple spaces after hash on atx style heading
null

(MD019, no-multiple-space-atx)